Formula/civl.rb

```diff
@@ -1,23 +1,37 @@
 class Civl < Formula
   desc "The Concurrency Intermediate Verification Language"
   homepage "https://vsl.cis.udel.edu/civl/"
-  url "https://vsl.cis.udel.edu/lib/sw/civl/1.7/r3157/release/CIVL-1.7_3157.tgz"
-  version "1.7-3157"
-  sha256 "49ed0467ea281bf5a436b2caf4f87862d3f613fa9e6e746ce52cfd409c3f4403"
+  url "https://vsl.cis.udel.edu/lib/sw/civl/1.20/r5259/release/CIVL-1.20_5259.tgz"
+  version "1.20-5259"
+  sha256 "15bf63b3a92300e8432e95397284e29aaa5897e405db9fc2d56cd086f9e330d3"
   revision 1
 
   bottle :unneeded
 
-  depends_on java: "1.8+"
+  depends_on "openjdk"
   depends_on "z3"
 
   def install
-    libexec.install "lib/civl-1.7_3157.jar"
-    bin.write_jar_script libexec/"civl-1.7_3157.jar", "civl"
+    underscored_version = version.to_s.tr("-", "_")
+    libexec.install "lib/civl-#{underscored_version}.jar"
+    bin.write_jar_script libexec/"civl-#{underscored_version}.jar", "civl"
     pkgshare.install "doc", "emacs", "examples", "licenses"
   end
 
   test do
+    ENV["HOME"] = testpath
```

Comment:
```suggestion
```

We don't need to do it explicitly, brew sets it automatically during the test run:
https://github.com/Homebrew/brew/blob/197c4666cf1b53fa3e59d97347fefac73bf88a98/Library/Homebrew/formula.rb#L1825